stacker/actions/destroy.py

```diff
@@ -83,6 +83,17 @@ def pre_run(self, outline=False, *args, **kwargs):
                 context=self.context
             )
 
+    def cleanup(self):
+        """Cleans up any resources that do not need to exist after stacker
+        has finished building all the stacks.
+
+        This is specifically used when deleting the sqs queues used for
+        polling events.
+
+        """
+        logger.debug('Cleaning up sqs queues')
```

Comment:
This isn't actually doing anything - the provider is doing the cleaning up. At most I'd say `Cleaning up` in here, but that's it.